app/utils.py

```diff
@@ -127,12 +127,36 @@ def email_address_is_nhs(email_address):
     ))
 
 
-def update_dct_to_str(update_dct):
-    str = "\n"
+def update_dct_to_str(update_dct, lang):
+    if lang not in ['EN', 'FR']:
+        raise NotImplementedError
+
+    values = {
+        'EN': {
+            'password': 'password',
+            'name': 'name',
+            'email_address': 'email address',
+            'mobile_number': 'mobile number',
+            'auth_type': 'auth type',
+        },
+        'FR': {
+            'password': 'mot de passe',
+            'name': 'nom complet',
+            'email_address': 'adresse courriel',
+            'mobile_number': 'téléphone cellulaire',
+            'auth_type': "méthode d'authentification",
+        }
+    }
+
+    content = "\n"
     for key in update_dct:
-        str += "- {}".format(key.replace("_", " "))
-        str += "\n"
-    return str
+        try:
+            key_name = values[lang][key]
+        except KeyError:
```

Comment:
My implementation is: have values in both languages that I'm almost certain they will be used after carefully looking at the code, but provide a fallback if I missed keys.

The fallback is English attribute name + replace `_` to `_` (`email_address` to `email address`). The previous code was only doing this part.